### DISCUSSION TITLE
Please update all available options for installation to the latest docs

### DISCUSSION THREAD

**Discussion by kolinfluence** [cortezaproject/corteza-server#1721] [General]
i googled and found broken installation file downloads for installation.

Can anyone provide the fastest and latest way to install with either docker or just pure github codes?

**Comment by kelanik8** [comment]
Hello @kolinfluence ,
You can follow this guide in the [docs](https://docs.cortezaproject.org/corteza-docs/2023.3/devops-guide/index.html#deploy-offline)
